ui/src/publication/card/publication-list-row.tsx

```diff
@@ -143,10 +143,14 @@ export const PublicationListRow: React.FC<PublicationListRowProps> = ({
                         })()}
                     </span>
                 </span>
-                {design && (
-                    <span className={styles['publication-list-item__design-name']}>{design}: </span>
-                )}
-                <span>{publication.message}</span>
+                <span>
+                    {design && (
+                        <span className={styles['publication-list-item__design-name']}>
+                            {`${design}:`}
+                        </span>
+                    )}
+                    {publication.message}
```

Comment:
Varsinaisen tiketin 🥩 . Aiempi versio rivittyi jännästi, sillä nuo rinnakkaiset `<span>`:it olivat flexboxin lapsia